library_generation/new_client/new-client.py

```diff
@@ -0,0 +1,458 @@
+    print(f"Generating from {versioned_proto_path}")
+    # parse BUILD.bazel in proto_path
+    client_input = parse(
+        build_path=Path(f"{sys.path[0]}/../../output/{versioned_proto_path}")
+        .resolve(),
+        versioned_path=versioned_proto_path,
+    )
+    repo_root_dir = Path(f"{sys.path[0]}/../../").resolve()
+    # run generate_library.sh
+    subprocess.check_call([
+        "library_generation/generate_library.sh",
+        "-p",
+        versioned_proto_path,
+        "-d",
+        destination_path,
+        "--gapic_generator_version",
+        generator_version,
+        "--protobuf_version",
+        "23.2",
+        "--proto_only",
+        client_input.proto_only,
+        "--gapic_additional_protos",
+        client_input.additional_protos,
+        "--transport",
+        client_input.transport,
+        "--rest_numeric_enums",
+        client_input.rest_numeric_enum,
+        "--gapic_yaml",
+        client_input.gapic_yaml,
+        "--service_config",
+        client_input.service_config,
+        "--service_yaml",
+        client_input.service_yaml,
+        "--include_samples",
+        client_input.include_samples,
+        "--versions_file",
+        f"{repo_root_dir}/versions.txt"],
+        cwd=repo_root_dir
+    )
+
+    # Move generated module to repo root.
+    __move_modules(
+        source=output_dir,
+        dest=repo_root_dir,
+        name_prefix="java-"
+    )
+
+    # Repo level post process
+    script_dir = "library_generation/repo-level-postprocess"
+    print("Remove irrelevant files from templates")
+    subprocess.check_call(
+        [
+            "bash",
+            f"{script_dir}/update_owlbot_postprocessor_config.sh"
+         ],
+        cwd=repo_root_dir
+    )
+
+    print("Deleting non generated samples")
+    subprocess.check_call(
+        [
+            "bash",
+            f"{script_dir}/delete_non_generated_samples.sh"
+        ],
+        cwd=repo_root_dir
+    )
+
+    print("Regenerating root pom.xml")
+    subprocess.check_call(
+        [
+            f"{script_dir}/generate_root_pom.sh",
+            f"{output_dir}"
+        ],
+        cwd=repo_root_dir,
+    )
+
+    print("Consolidating configurations")
+    subprocess.check_call(
+        [
+            f"{script_dir}/consolidate_config.sh",
+            f"{output_dir}"
+        ],
+        cwd=repo_root_dir,
+    )
+
+    if not split_repo:
+        print("Regenerating the GAPIC BOM")
+        subprocess.check_call(
+            [
+                f"{script_dir}/generate_gapic_bom.sh",
+                f"{output_dir}"
+            ],
+            cwd=repo_root_dir,
+        )
+
+        print("Setting parent poms")
+        subprocess.check_call(
+            [
+                "bash",
+                f"{script_dir}/set_parent_pom.sh",
+            ],
+            cwd=repo_root_dir,
+        )
+
+    print("Applying the versions")
+    subprocess.check_call(
```

Comment:
As I mentioned in another [comment](https://github.com/googleapis/sdk-platform-java/pull/2333/files#r1461254257), can we remove this script?

Reply:
Removed